front/src/views/serials/serials.view.html

```diff
@@ -1,3 +1,27 @@
-<div>    
-  This is a serial page.
+<div class="Wrap">    
+  <el-row class="el-row-wrap" :gutter="20" type="flex" justify="start">
+    <el-col v-for="serial in getSerialsList" class="grid-col-wrap" :key = "serial.serialId">
+      <div class="grid-content-wrap">
+        <el-image fit='cover' :src="serial.posterImageSource | getImgSrc(baseUrl)"></el-image>
```

Comment:
just `serial.posterImageSource | imageServerPath`
baseUrl should be added in filter from config